src/main/java/net/dv8tion/jda/api/JDA.java

```diff
@@ -1748,47 +1748,62 @@ default List<RichCustomEmoji> getEmojisByName(@Nonnull String name, boolean igno
         return getEmojiCache().getElementsByName(name, ignoreCase);
     }
 
+    /**
+     * Creates a new {@link ApplicationEmoji} for this JDA.
+     *
+     * <p>Note that a JDA is limited to 2000 normal/animated emojis.
```

Comment:
The "a JDA" sounds odd. And this info is also a bit wrong, given a bot can have multiple JDA instances through a ShardManager.
So it would be better to say something similar to `<p>Note that the bot is limited to 2000 Application Emojis (normal and animated).`

Reply:
Thank for the idea.